walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -118,4 +133,132 @@ else if(tr.contains(currentMinTime)) {
     }
     return openTimeRanges;
   }
+
+  /**
```

Comment:
Awesome javadocs! Way to keep it detailed with examples :)

Reply:
Thanks!